tsconfig.json

```diff
@@ -22,7 +22,7 @@
       "react-modal-hook": ["node_modules/@concord-consortium/react-modal-hook/dist"],
       "mobx-state-tree": ["node_modules/@concord-consortium/mobx-state-tree/dist"]
     },
-    "types": ["w3c-web-serial"]
+    "types": ["w3c-web-serial","node"]
```

Comment:
Adding node was needed in order to use node's file system module (fs) in the update script. I'm not sure it should be a permanent change since it's only needed when the script is being run, which will be a rare occurrence.

Should I not include this change?

Reply:
@kswenson Thanks -- good to know. I've made that change.